examples/static_react_task_with_tips/README.md

```diff
@@ -0,0 +1,131 @@
+# Simple Static React Task With Tips
```

Comment:
I copied the readme of static_react_task for this except for the new `Adding Tips` section.